tests/test_interface.rb

```diff
@@ -1444,6 +1417,7 @@ def test_ipv4_pim_sparse_mode
     #   interface Ethernet1/1
     #     ip pim sparse-mode
     #
+    i.switchport_mode = :disabled
```

Comment:
The test seems to pass for n7k:

```bash
TestInterface#test_ipv4_pim_sparse_mode = 10.42 s = .
```

`interfaces[0]` grabs `eth3/1` as the interface to test against, so I'm not sure if we've coded around this issue or if my node just happens to have compatible interfaces.